Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -256,11 +257,16 @@ extension LoginDetailViewController: UITableViewDelegate {
         }
 
         // Menu actions for Website
-        else if item == .WebsiteItem {
+        if item == .WebsiteItem {
             return action == MenuHelper.SelectorCopy || action == MenuHelper.SelectorOpenAndFill
-        } else {
-            return false
         }
+
+        // Menu actions for Username
+        if item == .WebsiteItem {
```

Comment:
Turns out that the reason `canPerformAction` wasn't being called is because `LoginTableViewCell`'s `canPerformAction` was being called instead. It has the same logic as the the table view's `canPerformAction` so all of that code inside the cell is not needed. Removing that properly calls the tableView's `canPerformAction`. Good catch :+1: